spec/features/views_show_page_spec.rb

```diff
@@ -25,14 +25,14 @@
       user: user_1,
       court: court,
       description: "Shot Callin",
-      start_time: Time.zone.now + 100000
+      start_time: Time.now + 100000
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.